src/diagapp.jl

```diff
@@ -33,26 +45,27 @@ function dfun(A, i, v)
 end
 
 # Calculating Diagonal Approximation for Inverse of Matrix
+"""
+    diagapp(A::AbstractArray)
+
+Diagonal Approximation algorithm for inverse of matrix (SPD low condition number).
+"""
 function diagapp(A)
+    return diagapp!(A) 
+end
+
+"""
+    diagapp!(A::AbstractArray)
+
+Diagonal Approximation algorithm for inverse of matrix (SPD low condition number).
+"""
+function diagapp!(A)
```

Comment:
Why the `!`? In Julia, this usually means that `A` is modified somehow, which it is not.

Reply:
Yes, I had some confusion for this one. Will fix that asap.